### PR TITLE
Deprecate r77 pylib usage

### PR DESCRIPTION
**Pull request by tahitihat**
Remove customized pylib dependencies to make utility scripts easier to run. 
- Use of `Field`/`Command` class for argument parsing ➡️ vanilla/out-of-box `argparse`
- Use of `FileUtils` ➡️ locally-defined utility functions 
- User of `TermColor` ➡️ unformatted log statement

### CODE REVIEW

File: scripts/__pycache__/__init__.cpython-39.pyc
Comment (Ideonvn) [inline scripts/__pycache__/__init__.cpython-39.pyc:1]: These files don't have to be in the repo, lets add [Python to .gitignore](https://github.com/github/gitignore/blob/main/Python.gitignore).

File: scripts/translations_util.py [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+        dir
+        and dir != '/'
+        and os.path.isdir(dir)
+        and not os.path.exists(os.path.join(dir, '.git'))
+    ):
+        dir = os.path.dirname(dir)
+    return dir
+
+
+def get_absolute_filepath(filename: str) -> str:
```
Comment (Ideonvn) [inline scripts/translations_util.py:19]: NITPICK: Typing can be updated here, you can pass in None and return None according to the logic.

```
Optional[str]
```

File: scripts/__pycache__/__init__.cpython-39.pyc
Comment (tahitihat) [reply scripts/__pycache__/__init__.cpython-39.pyc:1]: Oops, I did this on main (222e9c3) but rebased after creating the PR

**Review by Ideonvn — APPROVED** [review]

File: scripts/translations_util.py [hunk 2]
```diff
@@ -1,6 +1,36 @@
...
+
+def get_absolute_filepath(filename: str) -> Optional[str]:
+    """Return the absolute path for the filename, if it exists."""
+    if not filename:
+        return None
+
+    if os.path.exists(filename):
+        return os.path.normpath(os.path.abspath(filename))
+
+    if filename[0] == '/':
```
Comment (Sybrand) [inline scripts/translations_util.py:27]: If the file doesn't exist - shouldn't we be throwing an exception instead? How are we finding the absolute path of a non-existent file?

File: scripts/translations_util.py [hunk 3]
```diff
@@ -12,7 +42,7 @@ def get_translation_files(exclude_root: bool = False) -> List[str]:
     Returns:
         List[str]
     '''
-    root_dir = '%s/web/client' % FileUtils.GetSrcRoot()
+    root_dir = '%s/web/client' % find_src_root()
```
Comment (Sybrand) [inline scripts/translations_util.py:45]: f-strings?

File: scripts/translations_watch.py [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 
 I18N_ROOT = 'web/client'
-SRC_ROOT = FileUtils.GetSrcRoot()
-TRANSLATIONS_MAIN = FileUtils.GetAbsPathForFile('scripts/watcher/main.js')
+SRC_ROOT = find_src_root()
+TRANSLATIONS_MAIN = get_absolute_filepath('scripts/watcher/main.js')
 
 
-def translations_watch(verbose: bool = False) -> None:
+def translations_watch(args) -> None:
```
Comment (Sybrand) [inline scripts/translations_watch.py:11]: Type?

File: scripts/cli_translations.py [hunk 5]
```diff
@@ -18,136 +18,134 @@
...
+        type=str,
+        required=True,
+        help="IDO code for new locale",
     )
+    add_locale_parser.set_defaults(func=translations_add_locale)
 
-    Command.initialize_commands()
-    Command.run(Flags.ARGS.command)
+    args = parser.parse_args()
+    print(args)
```
Comment (Sybrand) [inline scripts/cli_translations.py:143]: This prints out `Namespace(subcommand=None)` - I'm guessing this was you debugging something?

**Review by Sybrand — CHANGES_REQUESTED** [review]
Mostly minor comments - and minor change requests.

File: scripts/__pycache__/__init__.cpython-39.pyc
Comment (AbbyBeeler) [inline scripts/__pycache__/__init__.cpython-39.pyc:1]: Looks like these are still being tracked by github, despite the `scripts/__pycache__/` line in the .gitignore

Running `git rm -r --cached scripts/__pycache__/` should fix it

File: scripts/translations_util.py [hunk 6]
```diff
@@ -1,6 +1,36 @@
-from typing import List
+import os
 import subprocess
-from pylib.file.file_utils import FileUtils
+from typing import List, Optional
+
+
+def find_src_root() -> str:
+    '''Return the project source root directory.'''
+    dir = os.getcwd()
```
[the remark this answers is not in the document]
Comment (tahitihat) [reply scripts/translations_util.py:8]: I'm planning to work on this in the future, but it's out of scope for this PR — right now I just want to remove the weird pylib imports and have the scripts operate the same way they did using pylib.

File: scripts/translations_util.py (hunk 2 above)
Comment (tahitihat) [reply scripts/translations_util.py:27]: This is same signature that the previously imported utility used (ie returning None when the file doesn't exist) so my preference is to keep it as-is for the sake of consistency and restricting the scope of this PR. 
Two other notes
- Isn't the 'file doesn't exist' case essentially covered?
- I don't actually like the way that `get_absolute_filepath` is used so it's likely that I'll remove it altogether in the future

**Review by Sybrand — APPROVED** [review]

**Pull request merged**